readme.md

```diff
@@ -46,33 +46,31 @@ Returns a Promise that resolves *usually* nothing.
 
 #### input
 
-*Required*  
-Type: `string` or `array`
+Type: `string` `Array`
 
 Either a string with the command or an array with the command (one argument per value).
 
 #### options
 
-Type: `object`
+Type: `Object`
 
-Options to be passed on to [`child_process.execFile()`](https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback).
+Options to be passed on to [`execa`](https://github.com/sindresorhus/execa#execafile-arguments-options).
 
-### nircmd.spawn(input, options)
+### nircmd.spawn(input, [options])
```

Comment:
Now I remember, because of https://github.com/sindresorhus/trayballoon/blob/master/index.js#L51.